stylus/settings/palette.json

```diff
@@ -1,32 +1,33 @@
 {
-  "black": "#000000",
-  "white": "#FFFFFF",
-  "paleGrey": "#F5F6F7",
-  "silver": "#D6D8Da",
-  "coolGrey": "#95999D",
-  "slateGrey": "#5D6165",
-  "charcoalGrey": "#32363F",
-  "zircon": "#F1F7FF",
-  "frenchPass": "#C2DCFF",
-  "dodgerBlue": "#297EF2",
-  "scienceBlue": "#0B61D6",
-  "puertoRico": "#4DCEC5",
-  "emerald": "#35CE68",
-  "malachite": "#08b442",
-  "texasRose": "#FFAE5F",
-  "chablis": "#FFF2F2",
-  "yourPink": "#FDCBCB",
-  "pomegranate": "#F52D2D",
-  "monza": "#DD0505",
-  "portage": "#9169F2",
-  "azure": "#1FA8F1",
-  "melon": "#FD7461",
-  "blazeOrange": "#FC6D00",
-  "mango": "#FF962F",
-  "pumpkinOrange": "#FF7F1B",
-  "darkPeriwinkle": "#6984CE",
-  "purpley": "#7F6BEE",
-  "lightishPurple": "#B449E7",
-  "barney": "#922BC2",
-  "weirdGreen": "#40DE8E"
+  "black": "var(--black)",
```

Comment:
Since this file is now referencing css variables, can we still use it in js?

Reply:
I guess so yes. Usually you just pick the color with JS and add it with styles, like inline-style. If so then there's no problem, but is there other usecases you can think of?

Reply:
We're using them for [`fill`](https://github.com/cozy/cozy-ui/blob/master/react/Icon/index.jsx#L39) values too, but I think that will also work with css variables? I think that's actually [the only use case](https://github.com/search?p=1&q=org%3Acozy+settings%2Fpalette.json&type=Code) 👍 .

Reply:
`fill` attribute or CSS property ?